src/java/org/apache/cassandra/cql3/statements/UpdateStatement.java

```diff
@@ -125,7 +125,8 @@ public void addUpdateForKey(PartitionUpdate.Builder updateBuilder, Clustering<?>
                 updates.get(i).execute(updateBuilder.partitionKey(), params);
 
             Row row = params.buildRow();
-            evaluateConstraintsForRow(row, metadata);
+            if (params.options.areConstraintsEnabled())
```

Comment:
If we are going this route (which I think it is the correct way to go), should we take the opportunity to expose this as part of the config yamls?